x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_tag_badge.tsx

```diff
@@ -76,9 +81,22 @@ export const RuleTagBadge = (props: RuleTagBadgeProps) => {
       )),
     [tags, tagItemDataTestSubj]
   );
+  if ('spread' in props) {
```

Comment:
nitpick: I think we should just destructure the spread instead of doing a property check 
```
const { spread = false } = props;
...
if (spread) {
  ...
}
```